packages/validator/src/services/indices.ts

```diff
@@ -110,6 +116,48 @@ export class IndicesService {
         newIndices.push(validatorState.index);
       }
     }
+
+    logValidatorStatuses(allValidators, this.logger);
+
     return newIndices;
   }
 }
+
+function logValidatorStatuses(allValidators: Map<ValidatorStatus, ValidatorResponse[]>, logger: Logger): void {
+  allValidators.get("active")?.forEach((validatorState: ValidatorResponse) => {
+    logger.info("Validator activated", {
+      pubkey: toHexString(validatorState.validator.pubkey),
+      index: validatorState.index,
+    });
+  });
+
+  [...(allValidators.get("pending_initialized") ?? []), ...(allValidators.get("pending_queued") ?? [])]?.forEach(
+    (validatorState: ValidatorResponse) => {
+      logger.info("Validator pending", {
+        pubkey: toHexString(validatorState.validator.pubkey),
+        index: validatorState.index,
+      });
+    }
+  );
+
+  allValidators.get("withdrawal_done")?.forEach((validatorState: ValidatorResponse) => {
+    logger.info("Validator withdrawn", {
+      pubkey: toHexString(validatorState.validator.pubkey),
+      index: validatorState.index,
+    });
+  });
+
+  [...(allValidators.get("exited_slashed") ?? []), ...(allValidators.get("exited_unslashed") ?? [])]?.forEach(
+    (validatorState: ValidatorResponse) => {
+      logger.info("Validator Exited", {
+        pubkey: toHexString(validatorState.validator.pubkey),
+        index: validatorState.index,
+      });
+    }
+  );
+
+  logger.info("Initialized Validators: ", {
+    total: allValidators.size,
+    available: allValidators.get("active")?.length,
```

Comment:
why is the term `available` used here instead of `active`? IMO this should be called `active`

```suggestion
    active: allValidators.get("active")?.length,
```